### ISSUE TITLE
update for IOS9.3.3

### ISSUE THREAD

**Issue by ddqp** [H6nry/ncforward#4]
hi  can NCforward be updated for IOS9.3.3?

**Comment by H6nry** [comment]
Yes, do not hesistate to fork NCForward and update it to iOS 9! I have no iOS 9 device at the moment, so no update from me. Sorry.

Am 03.08.2016 um 05:28 schrieb ddqp notifications@github.com:

**Comment by ddqp** [comment]
me again, replied to you on twitter. just let other people see my solution.
1. install NCFORWARD
2. go to settings, disable it, respring
3. setup host ip and pushbullet
4. respring
5. it works

it's a really the only app I found now to mirror iOS notification to other devices (pc, mac, etc), pushbullet does not work with my iphone and mac. so thanks a lot, just hope that you could get a chance to update this project again, maybe you can release a paid version to get a IOS9 device.
